tests/test_level/test_level_04.py

```diff
@@ -35,6 +35,22 @@ def test_print_single_quoted_text(self):
             max_level=11,
             expected=expected)
 
+    def test_print_microbit(self):
+        code = "print 'a'"
+        expected = textwrap.dedent(f"""\
+            from microbit import *
+            while True:
+                display.scroll('a')""")
+
+        self.multi_level_tester(
+            code=code,
+            translate=False,
+            skip_faulty=False,
+            expected=expected,
+            max_level=11,  # teodor, set this to 17, see the test fail and moke sure that it then does not fail anymore!
```

Comment:
We need to remove this comment, and also do we need to make it into 17 and update the code (either in this PR or in a separate one)